tests/org.eclipse.n4js.lang.tests/src/org/eclipse/n4js/naming/QualifiedNameComputerTest.java

```diff
@@ -43,7 +42,7 @@
 	private Type type;
 
 	@Inject
-	QualifiedNameComputer qnc;
+	ResourceNameComputer qnc;
```

Comment:
field name

ALSO: this class should be renamed (`QualifiedNameComputerTest` is obsolete after removing the `QualifiedNameComputer`)

Reply:
I did rename the field, but I think is ok, especially that it tests the conversion to JS identifiers (the Unicode representation of problematic characters)

Reply:
I meant: class should be renamed to ResourceNameComputerTest.